### ISSUE TITLE
[TC_AUTHOR_36 > Enable course highlight emails ] Emails are not coming after enabling it

### ISSUE THREAD

**Issue by fayyazahmed66** [openedx/build-test-release-wg#220]
**Steps to reproduce:**

1. Go to the studio with valid admin credentials https://studio.olive.demo.overhang.io/course/course-v1:edly+edly0001+2022
2. enable course highlight emails
3. add highlights in the course section
4. check if any email received to the learner

**Expected result:**
As per flow, the learner must receive highlights emails after enabling the feature

**Actual result:**
Emails are Not coming.

**Comment by fayyazahmed66** [comment]
label: olive testing

**Comment by mariajgrimaldi** [comment]
Emails were being sent to the spam folder before, can you test again to see if this is working? Also, [the docs](https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/manage_live_course/automatic_email.html#course-highlight-email) say the emails are sent depending on the section due date, was this considered during the setup?

**Comment by fayyazahmed66** [comment]
@mariajgrimaldi  let me test it again by following your instructions and document. I will update you here.

**Comment by DeanJayMathew** [comment]
label: olive release blocker

**Comment by DeanJayMathew** [comment]
label: release blocker

**Comment by mariajgrimaldi** [comment]
Let me know if you need help with this :)

**Comment by mariajgrimaldi** [comment]
This is a setup issue. First, the _official_ documentation is out of date. Second, it's complex to setup. After hitting my head against a wall for quite some time, I found [this doc](https://docs.google.com/document/d/19NF8YX_D_LQRHamfxYOKkSzMHu_kZb1xKGA4wFq08jk/) that Jill at Opencraft attached to [a post](https://discuss.openedx.org/t/weekly-course-highlight-messages/1173/3). They mentioned we need system admins' permissions for this to work, given that we need to run a command so the emails are sent. This raises an interesting question, what should we do with tests that need system admins' intervention? That's not the only issue, the setup must be proper for the course highlights to work.

**Comment by arbrandes** [comment]
@mariajgrimaldi, so, it looks like we'll need @regisb's help to try and enable this in the demo server, right?

**Comment by mariajgrimaldi** [comment]
@arbrandes: right! And thanks for removing the label! I also say this is not a blocker since it didn't fail. Instead, it's missing the proper setup.

**Comment by regisb** [comment]
@arbrandes  Surely you meant to say that someone should write a plugin to enable course highlights, right? :stuck_out_tongue: In all seriousness, this is a feature that requires too much work to set up in core or the demo server. It's not like we need to enable a waffle flag or two. It's a shame because it looks like a great feature, but the required setup is just too complicated.

For instance:

1. Why does it default to sending emails with sailthru and not the common Django mailer configuration?
2. Why use cron jobs when we could use `celery beat`?
3. Why do we need a site configuration, waffle switch AND waffle flag to enable this feature?

**Comment by arbrandes** [comment]
Makes sense, @regisb. :)  Demoted to "enhancement".

Also letting you off the hook, @mariajgrimaldi. ;)

**Comment by angonz** [comment]
Hi all!
We have created this [tutor plugin that enables highlights and nudge emails](https://github.com/aulasneo/tutor-contrib-nudge). Hope it helps! Currently it only supports K8s. Maybe someone can volunteer and enhance it for local.
